pkg/util/binding.go

```diff
@@ -42,8 +39,8 @@ func IsBindingReplicasChanged(bindingSpec *workv1alpha2.ResourceBindingSpec, str
 // GetSumOfReplicas will get the sum of replicas in target clusters
 func GetSumOfReplicas(clusters []workv1alpha2.TargetCluster) int32 {
 	replicasSum := int32(0)
-	for i := range clusters {
-		replicasSum += clusters[i].Replicas
+	for _, targetCluster := range clusters {
+		replicasSum += targetCluster.Replicas
```

Comment:
We use `for i := range clusters` in order to avoid copying the `workv1alpha2.TargetCluster` in this loop.

Reply:
The problem may not occur in all cases. For details, can see here: https://github.com/golang/go/wiki/CommonMistakes#using-reference-to-loop-iterator-variable